truss/truss/truss.go

```diff
@@ -1,5 +1,6 @@
 // Package truss contains the relative file tree data structure that represents
 // the paths and contents of generated files
+
```

Comment:
Additionally, and not specific to this reivew, we must be mindful of how we're writing our documentation comments. For example, here's what I get when I ask for documentation on the `truss` package:

```
➜  godoc github.com/TuneLab/go-truss/truss/truss
use 'godoc cmd/github.com/TuneLab/go-truss/truss/truss' for documentation on the github.com/TuneLab/go-truss/truss/truss command

PACKAGE DOCUMENTATION

package truss
    import "github.com/TuneLab/go-truss/truss/truss"


TYPES

type NamedReadWriter interface {
    io.ReadWriter
    Name() string
}
    NamedReadWriter represents a file name and that file's content Name() is
    a path relative to the directory containing the .proto files Name()
    should start with "service/" for all generated and read in files os.file
    fulfills the NamedReadWriter interface
```
That `NamedReadWriter` is a glob of text that muddies things, not clarifies things.